### DISCUSSION TITLE
Hierarchical Planning

### DISCUSSION THREAD

**Discussion by IfrahIdrees** [JuliaPlanners/PDDL.jl#17] [Q&A]
Hello, 

Thank you for creating and maintaining the package!

I was wondering if PDDL.jl supports Hierarchical Task Planning and if there are any developed examples that I can experiment with?

Thank you!

**Comment by ztangent** [comment]
You're welcome!

Unfortunately PDDL.jl does not currently support hierarchical planning, but I may work on a extension library that supports [HDDL](https://ojs.aaai.org/index.php/AAAI/article/view/6542) in the future.

**Comment by ztangent** [comment]
Converting this issue to a discussion, since this won't be fixed in this particular package!
